docs/user/icl/ssh-add-key-to-artemis.rst

```diff
@@ -0,0 +1,66 @@
+.. _use ssh key:
+
+Using SSH with Artemis
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Content of this document
+    :local:
+    :depth: 1
+
+You can use SSH keys to establish a secure connection between your computer and Artemis when you are performing Git operations (pull, clone, push) from your local machine.
+Personal keys are attached to your Artemis account – they are bound by that account's permissions and use the account's identity for any operations.
+To use your generated SSH keys with Artemis, you need to add it in the account Settings.
+
+
+Add an SSH key to your Artemis account
+""""""""""""""""""""""""""""""""""""""
+
+**1. Copy your public key**
+
+On Windows in your command prompt, change directory to your .ssh directory, and copy the public key file to your clipboard by running:
```

Comment:
```suggestion
On Windows in your command prompt, change directory to your `.ssh` directory, and copy the public key file to your clipboard by running:
```

Reply:
I'm also not sure if that looks better here, I think it hinders the flow of reading

Reply:
Then let's make it italics maybe? I really don't like technical terms/exact names in a text without highlighting them